indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamIndexTaskRunner.java

```diff
@@ -534,6 +534,7 @@ public void run()
                 } else {
                   rows = new ArrayList<>();
                   for (byte[] valueBytes : valueBytess) {
+                    log.info("valueBytes len: " + valueBytes.length);
```

Comment:
Is this really an `info` level log?

Reply:
Oh, it was my debugging log. Will remove.